man/as.matrix.Rd

```diff
@@ -11,30 +11,37 @@ of the columns in the \code{data.table} as the \code{matrix} \code{rownames}.
 
 \arguments{
 \item{x}{a \code{data.table}}
-\item{rownames}{optional, a single column name or column index to use as 
-the \code{rownames} in the returned \code{matrix}. If \code{TRUE} the
-\code{\link{key}} of the \code{data.table} will be used if it is a 
-single column, otherwise the first column in the \code{data.table} will
-be used. Alternative a vector of length \code{nrow(x)} to assign as the
-row names of the returned \code{matrix}.}
+\item{rownames}{optional, one of the following specifications of the row names:
+\enumerate{
+\item If \code{length(rownames) == length(nrow(x))} then \code{rownames} itself
```

Comment:
not sure but it might not be possible to use `\item` inside `\item{}{... \enumerate{.}}`. Probably best move to lower more descriptive sections.

Reply:
They can be definitely be nested. I separated them so that I could emphasize the first point later. Too OTT?

Reply:
so maybe incorrect number/placement of `{` somewhere

Reply:
Ah thanks!